### DISCUSSION TITLE
I can't enable claims.

### DISCUSSION THREAD

**Discussion by hxtchett** [GriefPrevention/GriefPrevention#1134] [Q&A]
### What happened:
_Land claims are disabled in this world._

### What was expected:
_A claim kit added to my inventory._

### Steps to reproduce:
__This may include a build schematic, a video, or detailed instructions to help reconstruct the issue__
1. Entering /griefprevention:claim or /griefprevention:createclaim.
2. Both commands come out with the same red message, "Land claims are disabled in this world."

### Server and GriefPrevention version:
___This can be found by running `/version` and `/version GriefPrevention` on your server.___
Paste of `/version`: This server is running Paper version git-Paper-261 (MC: 1.16.4) (Implementing API version 1.16.4-R01-SNAPSHOT)
Checking version, please wait...
You are 57 version(s) behind.
Paste of `/version GriefPrevention`: GriefPrevention version 16.12.0-d4d7ac4

### Stack trace/error or server log

```
Paste entire stack trace right here inside these backticks.

```
Or, paste the server.log at gist.github.com and paste the link here: 

### GriefPrevention config.yml

```yml

Paste the config.yml right here inside these backticks. Please do not use a paste service for this.

```

### Debugging steps attempted

- [ ] I attempted running only GriefPrevention on the server
  - [ ] Issue still occurs even if GriefPrevention is the only plugin running
- [ ] I attempted testing for the issue on a new server
  - [ ] Issue still occurs even after testing on a new server

**Comment by bigpresh** [comment]
Please share your GP config so we can see what you've set up.

It could be as simple as not actually enabling GP for the world(s) you want it active in under Claims.Mode.

You  need something like:

```
  Claims:
    Mode:
      world: Survival
```

... which means that the world named `world` will have GP active with the `Survival` rules.
